### ISSUE TITLE
"TypeError: Load Failed" on IOS

### ISSUE THREAD

**Issue by kamransoomro84** [mProjectsCode/obsidian-media-db-plugin#135]
-   [x] The Plugin is up to date
-   [x] Obsidian is up to date

**Describe the bug**

I've just installed the plugin. When I try to add an entry, I get the above error. I have provided an OMDB api key that I have confirmed works. 

**To Reproduce**

Search for a movie or show. 

**Expected behavior**
The media entry should be created in the appropriate folder. 

**Occurs on**

-   [ ] Windows
-   [ ] macOS
-   [ ] Linux
-   [ ] Android
-   [x] iOS

**Plugin version**
0.6.0

**Comment by mProjectsCode** [comment]
Does the same happen with all other plugins disabled?

Can you somehow open the dev console and copy the error message displayed there?

**Comment by kamransoomro84** [comment]
Yes it still happens with all plugins disabled. I can also confirm that it works on Mac. 

How do I open the developer console on iOS?

**Comment by mProjectsCode** [comment]
I don't know. I don't own an Apple device.

**Comment by kamransoomro84** [comment]
I tried by installing Obsidian Dev Tools. 

This is Al I see in the console:

![image](https://github.com/mProjectsCode/obsidian-media-db-plugin/assets/4132530/0b07e875-dd7b-46c6-96f0-a2f365a55453)

**Comment by cholston** [comment]
I'm having the same issue. I can confirm that it used to work, but I'm not sure when it broke.

**Comment by StephBoyardee** [comment]
Ditto I have run into the same issue

**Comment by kamransoomro84** [comment]
Any ETA for when this might be fixed?

**Comment by mProjectsCode** [comment]
i can try to take a look tomorrow

**Comment by ScotchNebular** [comment]
Same issue for me. Used to work on iOS, but no longer does. I have no idea when it stopped. And it works fine on Mac desktop.

**Comment by leifg** [comment]
Same issue for me

**Comment by mProjectsCode** [comment]
Not sure why this is happening and as stated above, I don't own an Apple device to debug.

**Comment by BrianBatchelder** [comment]
This is no longer happening to me.

**Comment by kamransoomro84** [comment]
I can confirm that it's not happening for me either. However, I am now getting weird, completely unrelated results when I search for something. 

For example, this is what I get if I search for `top gun`. 

![image](https://github.com/mProjectsCode/obsidian-media-db-plugin/assets/4132530/d9149a2f-a35d-4ec4-902a-c78da40cf03c)

**Comment by BrianBatchelder** [comment]
Those results look to me like the OMDB API is broken. Try upgrading the media db plugin to the latest version. It fixes a problem using the API.

**Comment by kamransoomro84** [comment]
> Those results look to me like the OMDB API is broken. Try upgrading the media db plugin to the latest version. It fixes a problem using the API.

That worked! Thanks!

**Comment by mProjectsCode** [comment]
This seems to be solved
